modules/running-cluster-oci-c3-agent-based.adoc

```diff
@@ -0,0 +1,35 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-agent-based-installer.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="running-cluster-oci-c3-agent-based_{context}"]
+= Running a cluster on {oci-c3-short}
+
+To run a cluster on {oci-c3}, you must convert your generated Agent ISO image into an {oci} image, and then upload the image to the {oci-c3-short} system.
+
+// Everything below this comment has been copied from a similar module for the OCI procedure. I am expecting and ready to update any of this content, including links to Oracle docs.
```

Comment:
Highlighting this comment that I made here. Expecting to have to update the links for C3 specific instructions about taking the Agent ISO and starting to install the cluster